lib/Reader/abisignature.cpp

```diff
@@ -312,7 +312,7 @@ CallSite ABICallSignature::emitUnmanagedCall(GenIR &Reader, Value *Target,
 
 Value *ABICallSignature::emitCall(GenIR &Reader, Value *Target, bool MayThrow,
                                   ArrayRef<Value *> Args,
-                                  Value *IndirectionCell,
+                                  Value *IndirectionCell, bool IsJmp,
```

Comment:
Okay, that seems fine. You're right about the restrictions--I had misread the lang ref.